examples/rectangle.pln

```diff
@@ -0,0 +1,10 @@
+-- The following program demonstrates using table types in Pallene.
```

Comment:
Please add the copyright notice to all files.

Reply:
Okay, done.